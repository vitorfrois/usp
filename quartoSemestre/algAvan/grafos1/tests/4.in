10 20 10
6 9 8
4 7 1
9 10 8
5 6 7
4 9 2
1 2 5
8 9 8
6 8 8
7 9 2
6 10 1
1 4 6
2 3 2
4 5 7
1 5 5
3 4 7
7 8 1
6 7 8
7 10 3
8 10 7
5 10 3
1 2
8 4
9 10
4 6
10 4
4 5
9 10
1 7
7 8
8 4
