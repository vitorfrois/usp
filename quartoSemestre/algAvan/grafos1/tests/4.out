5
2
5
5
4
7
5
7
1
2
